frontend/js/src/year-in-music/2022/MagicShareButton.tsx

```diff
@@ -0,0 +1,197 @@
+import {
```

Comment:
not to be a party-pooper, but the exported function is `ShareButton` and the file is `MagicShareButton.tsx`